pkg/operator/configobservation/apiserver/observe_send_retry_after_while_not_ready_once.go

```diff
@@ -0,0 +1,76 @@
+package apiserver
+
+import (
+	"fmt"
+	"reflect"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/configobservation"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+const sendRetryAfterWhileNotReadyOnceKey = "send-retry-after-while-not-ready-once"
```

Comment:
I guess it's not possible, but can we make it static through config overriding or something? It should always be enabled for SNO.

Reply:
iiuc this would cause an additional rollout for SNO, but I'm not certain